### ISSUE TITLE
Quantization ops missing from benchmark program

### ISSUE THREAD

**Issue by kalle** [tensorflow/tensorflow#3979]
### Environment info
Operating System: Mac OS X 10.11.4
CUDA: No
TF installed from source
`git rev-parse HEAD`: `6d04d601e9e8758ec4642fa9d548b7321d804d63`
```
bazel version
Build label: 0.2.3-homebrew
Build target: bazel-out/local-fastbuild/bin/src/main/java/com/google/devtools/build/lib/bazel/BazelServer_deploy.jar
Build time: Tue May 17 15:07:52 2016 (1463497672)
Build timestamp: 1463497672
Build timestamp as int: 1463497672
```

### Problem
I want to run inference on mobile and thought quantization would be a good idea. I also want to use the [benchmark program](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/tools/benchmark) to see speed and memory consumption of different models before starting to train them.

However the benchmark seems to be missing the quantization ops needed (it doesn't matter which quantization mode I use):

```
E tensorflow/tools/benchmark/benchmark_model.cc:72] Could not create TensorFlow Session: Not found: Op type not registered 'QuantizeV2'
```

I realize that I should modify this [BUILD](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/benchmark/BUILD) file somehow to include the missing ops but I can't figure out Bazel's name of the built library or how to link it statically into the benchmark program. Or are the ops missing for a reason (incompatible with the normals ops etc)?

I know that this is more of a question than a bug report, but I don't think I'm the only one with problem so perhaps you can treat is as a documentation bug?

**Comment by lcwyylcwyy** [comment]
you can see this issue https://github.com/tensorflow/tensorflow/issues/3619
but it just solve the issue partly.

**Comment by prb12** [comment]
Closing as duplicate of #3619

**Comment by kalle** [comment]
@lcwyylcwyy Thanks, that did the trick!
